by-language/php-amphp/README.rst

```diff
@@ -0,0 +1,53 @@
+.. highlight: console
+
+###########################################################
+Connect to CrateDB and CrateDB Cloud using AMPHP/PostgreSQL
+###########################################################
+
+
+*****
+About
+*****
+
+This is an example program demonstrating CrateDB's `PostgreSQL wire protocol`_
+compatibility by exercising a few basic SQL statements using `AMPHP PostgreSQL
+driver`_, based on the `AMPHP`_ framework (see also `AMPHP on GitHub`_). It is
+an asynchronous PostgreSQL client for PHP.
+
+For further information, please also visit the `CrateDB clients and tools`_
+page. Because CrateDB only supports (implicitly created) `table schemas`_
+instead of databases, please also read `PostgreSQL documentation about
+schemas`_ carefully.
+
```

Comment:
Can we at least add a warning that the full functionally is not ensured and also not officially supported? Otherwise one may imply that this should completely work and also may expect that we provide full support for it.

Reply:
I've added a corresponding note to both README documents.

> Also, please note that support for this driver is experimental. The [CrateDB PDO driver](https://crate.io/docs/pdo/) is the official CrateDB PHP client supported by Crate.io.